lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -224,8 +224,8 @@ def add_files_to_build_phases(native_target, test_native_targets)
                               end
 
               headers = file_accessor.headers
-              public_headers = file_accessor.public_headers.map(&:realpath)
-              private_headers = file_accessor.private_headers.map(&:realpath)
+              public_headers = file_accessor.public_headers
+              private_headers = file_accessor.private_headers
```

Comment:
Mentioned this elsewhere but wondering if this would affect vendored frameworks which often use symlinks to point to different versions vs. "current"

Reply:
I change the get file list logic and it will list all files in symlink directory.